### ISSUE TITLE
executable code or not

### ISSUE THREAD

**Issue by KitWallace** [KitWallace/XQuery-Wikibook#9]
Pros and cons for having executable code

Pro: 
 It keeps us honest - I keep finding example code which has been posted which doesn't compile or doesn't work  
Its useful to see how a script works especially if it is interactive

Con The same code is in the exIstdb and in the wikibook and the versions will inevitably  drift apart. Likewise output has be be copied into the book.  An environment in which only one copy of the code exists  ( the exist demo and http://kitwallace.co.uk/Book/  using util:eval avoid this)
The code has to reside on a server somewhere (presntly on my personal server ) and this is not sustainable

**Comment by lguariento** [comment]
Uhm... is the code currently on your personal server? Are we looking for something more sustainable?

**Comment by KitWallace** [comment]
Yes that's correct - the server handles multiple websites so its not going away any time soon, but it is a longer term concern.

**Comment by KitWallace** [comment]
Duncan Paterson:  The XQuery Fiddle resource by Martin Honnen is perhaps more what people are used to coming from other languages. http://xqueryfiddle.liberty-development.net/  This is built on Saxonica's XQuery 3.1  and contains a large number of XQuey 3.1 tests
